app.py

```diff
@@ -19,26 +18,49 @@ def index():
     return {"message": data}
 
 
-
 @app.route('/recommend/popular')
 def popular_movies():
     data = recommend_popular_movies()
     return data
 
 
+@app.route('/html/popular')
+def popular():
```

Comment:
There is another redundant line of code that assigns a value to 'movie_name'. It can be removed.